Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php

```diff
@@ -22,7 +22,11 @@
 use Symfony\Component\HttpKernel\Log\LoggerInterface;
 use Symfony\Component\Translation\TranslatorInterface;
 
-class AdminExtractor implements ExtractorInterface, TranslatorInterface, SecurityHandlerInterface, LabelTranslatorStrategyInterface
+final class AdminExtractor implements
+    ExtractorInterface,
```

Comment:
> It's already 131 chars before the change, so no.
I don't see why there should be exceptions to the 120 rule (which is already pretty long, considering some coding standards like PEP-8 use less than 80 chars).

We are not following PEP-8. All class signature on those project are on-lined.

If you want to start a vote on dev-kit for that, feel free. For the moment, please keep it as is.

> You should review PRs commit by commit, by using the "Showing changes from all commits" select box above.

Reviewer maybe but a user looking for an issue/PR that can solve his problem would love to have a quick and easy preview of what is done without having to analyse the PR commit by commit! :+1: